hangman_lunch2.rb

```diff
@@ -0,0 +1,206 @@
+
+class Hangman
+
+  def initialize
+    @answer = populate_answer
+    # Displayed line to fill in correct guesses
+    @answer_line = answer_line
+    # count of wrong answers
+    @wrong_count = 0
+    @letter_array = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"]
+    @letters_guessed = []
+
+    start
+    draw_board
+    play
+  end
+
+  def start
+    puts "Welcome to Star Trek Hangman! Good luck!"
+    # Keeps track of game play
+    @now_playing = true
+  end
+
+# ------------------
+# Create answer
+  def populate_answer
+    answer_array = ["PICARD", "GUINAN", "WORF", "WARP", "ENGAGE", "ENTERPRISE", "DATA", "ROMULAN", "VULCAN", "BETAZOID", "KLINGON", "FEDERATION", "REPLICATOR", "DARMOK", "HOLODECK", "PHASER", "TRICORDER", "BEAM", "TRANSPORTER"]
+
+    answer = answer_array.shuffle.pop
+    return answer
+  end
+
+# ------------------
+# Visual methods
+
+# draws board depending on number of wrong answers
+  def draw_board
+    if @wrong_count == 0
+      puts """
+    ___________________          _-_
+    \\==============_=_/ ____.---'---`---.____
+                \\_ \\    \\----._________.----/
+                  \\ \\   /  /    `-_-'
+              __,--`.`-'..'-_
+             /____          ||
+                  `--.____,-'
+
+"""
+    elsif @wrong_count == 1
+      puts """
+                                 _-_
+                        ____.---'---`---.____
+                \\_ \\    \\----._________.----/
+                  \\ \\   /  /    `-_-'
+              __,--`.`-'..'-_
+             /____          ||
+                  `--.____,-'
+
+""".cyan
+    elsif @wrong_count == 2
+      puts """
+                                 _-_
+                        ____.---'---`---.____
+                   \\    \\----._________.----/
+                    \\   /  /    `-_-'
+              __,--`.`-'..'-_
+             /____          ||
+                  `--.____,-'
+
+      """.blue
+    elsif @wrong_count == 3
+      puts """
+                                _-_
+                       ____.---'---`---.____
+                       \\----._________.----/
+                        /  /   `-_-'
+              __,--`.`-'..'-_
+             /____          ||
+                  `--.____,-'
+
+      """.yellow
+    elsif @wrong_count == 4
+      puts """
+                                   _-_
+                          ____.---'---`---.____
+                          \\----._________.----/
+                          /  /    `-_-'
+
+
+
+
+
+      """.magenta
+    elsif @wrong_count == 5
+      puts """
+                                   _-_
+                          ____.---'---`---.____
+                          \\----._________.----/
+                                  `-_-'
+
+
+
+
+
+      """.red
+    else
+      lose
+    end
+
+  end
+
+# Creates blank answer line
+  def answer_line
+      @answer_line = []
+      (@answer.length).times do |ans|
```

Comment:
Ruby arrays have a shortcut of creating arrays of a certain size with default value by using its initializer:

`Array.new(@answer.length, ' _ ')`